crashreport/src/test/java/com/rakuten/tech/mobile/crash/AsyncHttpPostSpec.kt

```diff
@@ -0,0 +1,94 @@
+package com.rakuten.tech.mobile.crash
+
+import com.rakuten.tech.mobile.crash.AsyncHttpPost.CrashServerURL
+
+import java.net.HttpURLConnection
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.*
+import org.mockito.MockitoAnnotations
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.io.*
+import java.net.URLConnection
+
+/**
+ * Unit tests for AsyncHttpPost.java.
+ */
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = intArrayOf(27))
```

Comment:
Why is this an array instead of just `sdk = 27`? Also, could we move this to a `robolectric.properties` config file?

Reply:
I moved the robolectric config into a base class